sentry/src/main/java/io/sentry/EnvelopeSender.java

```diff
@@ -53,6 +57,14 @@ protected void processFile(final @NotNull File file, final @NotNull Hint hint) {
       return;
     }
 
+    if (envelopeCache.containsFile(file)) {
```

Comment:
`m`: I think we should do the same in `OutboxSender` as we also install the second instance of `SendCachedEnvelopeIntegration` for sending cached files from the outbox folder

Reply:
Also, a comment would be nice to mention why this is needed